src/graphql_qb/queries.clj

```diff
@@ -76,16 +76,25 @@
         (string/join "\n" and-clauses)))))
 
 (defn get-datasets-query
-  [dimensions measures uri configuration]
+  [dimensions measures uri configuration lang]
   (let [dataset-label (config/dataset-label configuration)]
     (str
       "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>"
       "PREFIX qb: <http://purl.org/linked-data/cube#>"
       "PREFIX dcterms: <http://purl.org/dc/terms/>"
-      "SELECT ?ds ?title ?description ?licence ?issued ?modified ?publisher WHERE {"
+      "SELECT ?ds ?name ?title ?description ?licence ?issued ?modified ?publisher WHERE {"
       (get-dimensions-or dimensions)
-      "  ?ds <" (str dataset-label) "> ?title ."
-      "  OPTIONAL { ?ds rdfs:comment ?description . }"
+      "  ?ds <" (str dataset-label) "> ?name ."
+      "  FILTER(LANG(?name) = '')"
+      "  OPTIONAL {"
+      "    ?ds <" (str dataset-label) "> ?title ."
```

Comment:
Are you using the ?name for labels without language and ?title for labels with language? It is not very clear to me

Reply:
Yes, name is used to generate parts of the GraphQL schema such as the dataset field name and associated type name. The literal label without a language tag is used for that purpose. Any labels with a language tag will be returned for the title field if that language is specified at the top of the the query e.g `cubiql(lang_preference: "fr") { dataset_earnings { title } }` would return the french label of the `Earnings` dataset.

Reply:
For datasets that use language tags I think we should not require also labels without a languge tag. This will create duplicate labels. In this case we could define a default language (e.g. at the configuration) that will be used for the schema of the GraphQL. What do you think?

Reply:
Yes that would be possible. Another is that we implement a basic version of #40 which would allow us to stop using labels to build the GraphQL schema. This approach would required us to generate extra data to specify the mapping (and will take longer to implement). Which would you prefer?

Reply:
I think it is better to do this at the configuration. Defining and using an ontology (even if it is basic) will take much longer.

Reply:
I've created #112 for this work so this PR can be merged along with #111.